src/lib.rs

```diff
@@ -434,61 +501,61 @@ make_color! {
     ///Linear luminance.
     Luma {
         ///Linear luminance.
-        y(luma: T)[alpha: T] => new;
+        y(luma: T)[alpha: T] => with_wp;
```

Comment:
Maybe we should favor D65 in this case, or add `*_with_wp` variants. Not sure what's best, but `Color` is meant to be easy to use.